tests/app/celery/test_nightly_tasks.py

```diff
@@ -232,34 +234,47 @@ def test_should_call_delete_inbound_sms(notify_api, mocker):
     assert nightly_tasks.delete_inbound_sms_older_than_retention.call_count == 1
 
 
-def test_create_ticket_if_letter_notifications_still_sending(mocker):
+def test_create_ticket_if_letter_notifications_still_sending(notify_api, mocker):
```

Comment:
How come we had to add the `notify_api` here?

Reply:
This test (and the other test that I added the fixture to) need an application context in order to run otherwise they give an error. Since the `notify_api` fixture is scoped to the session, these tests were passing when run as part of a group which uses the fixture but giving an error when run individually.

Reply:
Thank you, good spot.

If you were going to make this PR even better (don't worry, it's already brilliant), then that would be good to put in a separate commit explaining the change.

But definitely not worth doing now